backend/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/push/RuleSetChangedEvent.java

```diff
@@ -55,12 +56,19 @@ public static class ActiveRule {
     private final Map<String, String> parameters;
     private final String templateKey;
 
-    public ActiveRule(String key, String languageKey, IssueSeverity severity, Map<String, String> parameters, @Nullable String templateKey) {
+    private final List<RuleSetChangedEventParser.Impact> impacts;
+
+    public ActiveRule(String key, String languageKey, IssueSeverity severity, Map<String, String> parameters, @Nullable String templateKey, List<RuleSetChangedEventParser.Impact> impacts) {
       this.key = key;
       this.languageKey = languageKey;
       this.severity = severity;
       this.parameters = parameters;
       this.templateKey = templateKey;
+      this.impacts = impacts;
+    }
+
+    public List<RuleSetChangedEventParser.Impact> getOverriddenImpacts() {
+      return impacts;
```

Comment:
Niptick: more of a personal taste, not sure it's an established convention, but I try to order the fields and the getters the same way